.github/workflows/release.yml

```diff
@@ -32,18 +32,70 @@ jobs:
         run: cd main/ && GOOS=linux GOARCH=amd64 go build -o ~/main/bin/amd64-linux-openshiftplugin-${{ github.event.inputs.version }} .
 
       - name: Build openshift plugin darwin amd64
-          env:
-            GOPROXY: "https://proxy.golang.org"
-          run: cd main/ && GOOS=linux GOARCH=amd64 go build -o ~/main/bin/amd64-linux-openshiftplugin-${{ github.event.inputs.version }} .
+        env:
+          GOPROXY: "https://proxy.golang.org"
+        run: cd main/ && GOOS=darwin GOARCH=amd64 go build -o ~/main/bin/amd64-darwin-openshiftplugin-${{ github.event.inputs.version }} .
 
       - name: Build openshift plugin darwin arm64
-          env:
-            GOPROXY: "https://proxy.golang.org"
-          run: cd main/ && GOOS=linux GOARCH=amd64 go build -o ~/main/bin/amd64-linux-openshiftplugin-${{ github.event.inputs.version }} .
+        env:
+          GOPROXY: "https://proxy.golang.org"
+        run: cd main/ && GOOS=darwin GOARCH=arm64 go build -o ~/main/bin/arm64-darwin-openshiftplugin-${{ github.event.inputs.version }} .
 
       - name: release
         uses: ncipollo/release-action@v1
         with:
           artifacts: "~/main/bin/*"
           token: ${{ secrets.GITHUB_TOKEN }}
-          tag: ${{ github.event.inputs.version }}
\ No newline at end of file
+          tag: ${{ github.event.inputs.version }}
+
+      - name: Checkout crane-plugins
+        uses: actions/checkout@v2
+        with:
+          repository: ${{ github.repository_owner }}/crane-plugins
+          token: ${{ secrets.PLUGIN_RELEASE }}
+
+      - name: Updating index file and adding manifest
+        run: |
+          cat << EOF >> index.yml
+          OpenshiftPlugin-${{ github.event.inputs.version }}: https://github.com/${{ github.repository_owner }}/crane-plugins/raw/main/plugins/OpenshiftPlugin-${{ github.event.inputs.version }}.yml
+          EOF
+          cat << EOF >> plugins/OpenshiftPlugin-${{ github.event.inputs.version }}.yml
+          name: OpenshiftPlugin
+          shortDescription: OpenshiftPlugin
+          description: this is OpenshiftPlugin
```

Comment:
This is the field that needs to descrtibe what the plugin does, right? It probably needs a more complete blurb describing the resources it modifies, etc. A few paragraphs (for the to-be-written `crane plugin-manager info` command).

However, updating that may make more sense as a separate follow-on PR. We probably shouldn't hold up automation on it, especially since it's not currently printed out anywhere.